README.md

```diff
@@ -6,14 +6,15 @@
 
 Rust playback library.
 
- - Playback is handled by [cpal](https://github.com/RustAudio/cpal).
- - MP3 decoding is handled by [minimp3](https://github.com/lieff/minimp3).
+Playback is handled by [cpal](https://github.com/RustAudio/cpal). Format decoding can be handled either by [Symphonia](https://github.com/pdeljanov/Symphonia), or by format-specific decoders:
+
+ - MP3 decoding can be handled by [minimp3](https://github.com/lieff/minimp3) (but defaults to [Symphonia](https://github.com/pdeljanov/Symphonia)).
```

Comment:
You already have "can be handled" wording right above

Reply:
Any better now?